src/topology/algebra/module.lean

```diff
@@ -131,23 +131,23 @@ instance : has_add (β →L[α] γ) :=
 ⟨λ f g, ⟨f + g, continuous_add f.2 g.2⟩⟩
 
 @[simp] lemma add_apply : (f + g) x = f x + g x := rfl
-@[simp, move_cast] lemma coe_add : ((f + g) : β →ₗ[α] γ) = (f : β →ₗ[α] γ) + g := rfl
-@[simp, move_cast] lemma coe_add' : ((f + g) : β → γ) = (f : β → γ) + g := rfl
+@[simp, move_cast] lemma coe_add : (((f + g) : β →L[α] γ) : β →ₗ[α] γ) = (f : β →ₗ[α] γ) + g := rfl
+@[move_cast] lemma coe_add' : (((f + g) : β →L[α] γ) : β → γ) = (f : β → γ) + g := rfl
 
 instance : has_neg (β →L[α] γ) := ⟨λ f, ⟨-f, continuous_neg f.2⟩⟩
 
 @[simp] lemma neg_apply : (-f) x = - (f x) := rfl
 
-@[simp, move_cast] lemma coe_neg : ((-f) : β →ₗ[α] γ) = -(f : β →ₗ[α] γ) := rfl
-@[simp, move_cast] lemma coe_neg' : ((-f) : β → γ) = -(f : β → γ) := rfl
+@[simp, move_cast] lemma coe_neg : (((-f) : β →L[α] γ) : β →ₗ[α] γ) = -(f : β →ₗ[α] γ) := rfl
```

Comment:
Yes. The previous version did nothing: the left and right hand side were exactly the same, as Lean did insert the coercion at the wrong place. It was spotted by `sanity_check` since for this stupid version some typeclass is not necessary. I added enough type annotations to make sure that the statement is what it should had been from the start.

Reply:
I'm glad `sanity_check` is spotting mistakes like this!